lib/src/components/power_sensor/power_sensor.dart

```diff
@@ -11,25 +10,48 @@ typedef Current = GetCurrentResponse;
 abstract class PowerSensor extends Resource {
   static const Subtype subtype = Subtype(resourceNamespaceRDK, resourceTypeComponent, 'power_sensor');
 
-  /// Obtain the measurements/data specific to this [Sensor]
-  /// If a sensor is not configured to have a measurement or fails to read a piece of data, it will not appear in the readings dictionary.
+  /// Get the measurements or readings from this [PowerSensor].
+  /// If a sensor is not configured correctly or fails to read a
+  /// piece of data, it will not appear in the readings dictionary.
+  ///
+  /// ```
+  /// var readings = await myPowerSensor.readings();
+  /// ```
   Future<Map<String, dynamic>> readings({Map<String, dynamic>? extra});
 
-  /// Get the voltage (volts) and boolean isAC
+  /// Get the voltage (in volts) and whether the power is
```

Comment:
nit: in other comments, the units were not in parenthesis.

Reply:
```suggestion
  /// Get the voltage in volts, and whether the power is
```